images/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:stable as java8-builder
```

Comment:
I suggest using SHAs to avoid possible breaking commits to underlying base image. As Marcin always suggests,  aim for hermetic builds.

Reply:
agree, I have updated using date based tags for these as those are more human readable and stable